sample-applications/blazor-stocks/Client/Components/StocksChart/MainChart.razor

```diff
@@ -6,46 +6,128 @@
+                <StockChartNavigatorSelectMousewheel Zoom="@ChartMousewheelZoom.Both" Reverse="true" />
+            </StockChartNavigatorSelect>
+            <StockChartNavigatorHint Visible="true" Format="{0:d} - {1:d}" />
+
+        </StockChartNavigator>
+
+        <StockChartSeriesItems>
+            <StockChartSeries Type="@StockChartSeriesType.Candlestick" Data="@ChartData"
+                              OpenField="@( nameof(StockIntervalDetails.Open) )"
+                              CloseField="@( nameof(StockIntervalDetails.Close) )"
+                              HighField="@( nameof(StockIntervalDetails.High) )"
+                              LowField="@( nameof(StockIntervalDetails.Low) )"
+                              DownColor="#FF6358"
+                              Color="@GetSeriesColor(MainChartType)">
+            </StockChartSeries>
+            <StockChartSeries Type="@StockChartSeriesType.Column" Data="@ChartData"
+                              Field="@( nameof(StockIntervalDetails.Volume) )"
+                              CategoryField="@( nameof(StockIntervalDetails.Date) )"
+                              ColorField="@( nameof(StockIntervalDetails.ColumnColor) )"
+                              Aggregate="@ChartSeriesAggregate.Sum"
+                              Gap="0.75"
+                              Axis="Volume">
+            </StockChartSeries>
+        </StockChartSeriesItems>
+
+        <StockChartCategoryAxes>
+            <StockChartCategoryAxis Type="@ChartCategoryAxisType.Date">
+                <StockChartCategoryAxisCrosshair Visible="true">
+                    <StockChartCategoryAxisCrosshairTooltip Visible="true" />
+                </StockChartCategoryAxisCrosshair>
+                <StockChartCategoryAxisLabels>
+                    <StockChartCategoryAxisLabels Step="@LabelStep"></StockChartCategoryAxisLabels>
+                </StockChartCategoryAxisLabels>
+                <StockChartCategoryAxisMajorGridLines Visible="true" />
+                <StockChartCategoryAxisMinorGridLines Visible="false" />
+            </StockChartCategoryAxis>
+        </StockChartCategoryAxes>
+
+        <StockChartValueAxes>
+            <StockChartValueAxis>
+                <StockChartValueAxisCrosshair Visible="true">
+                    <StockChartValueAxisCrosshairTooltip Visible="true" />
+                </StockChartValueAxisCrosshair>
+                @*<StockChartValueAxisLabels Format="@( SelectedCurrency.Sign + "{0}" )"  />*@
+                <StockChartValueAxisMinorGridLines Visible="false" />
+            </StockChartValueAxis>
+            <StockChartValueAxis Name="Volume" Min="0" Max="@VolumeValueAxisMax()" Visible="false"></StockChartValueAxis>
+        </StockChartValueAxes>
+
+        <StockChartLegend Visible="false" />
+        <StockChartTooltip Visible="true" />
+
+    </TelerikStockChart>
+}
+else
+{
+    <TelerikChart Transitions="true" RenderAs="@RenderingMode.SVG" @ref="@ChartRef">
```

Comment:
This is the default values for `Transitions` and `RenderAs` - why they are listed explicitly?

Reply:
That is a great question, to which I have no reasonable answer :) https://github.com/telerik/blazor-ui/pull/47/commits/45ec0ec2e5d33a117dee3d87295b5582a815feba